weaviate/values.yaml

```diff
@@ -231,7 +231,12 @@ env:
 
   # Set RAFT cluster expected number of voter nodes at bootstrap
   RAFT_BOOTSTRAP_EXPECT: 1
-
+  # RAFT_HEARTBEAT_TIMEOUT: 1
+  # RAFT_ELECTION_TIMEOUT: 1
+  # Control how often raft checks if it should perform a snapshot
+  # RAFT_SNAPSHOT_INTERVAL: 120
+  # Number of outstanding log entries before performing a snapshot
+  # RAFT_SNAPSHOT_THRESHOLD: 8192
```

Comment:
Red, can you make appropriate comments on each env variable describing what it does and what's the default value?

We have a rule that we try to make descriptive comments on each env variable so that the user would know what a given setting does